templates/partials/feedback.html

```diff
@@ -0,0 +1,18 @@
+<div>
+    <h3>{{ feedback_title }}</h3>
```

Comment:
Do we need this as a variable? Is the title ever going to be anything other than "Feedback"?

Reply:
Good point.  Just had the variable for consistency with the others I guess.